src/Node/Expression/CallExpression.php

```diff
@@ -23,6 +23,9 @@ public function compile(Compiler $compiler): void
         $this->compileCallable($compiler);
     }
 
+    /**
+     * @return void
```

Comment:
This `@return` should be added in 3.x, to be able to add an actual return type in 4.x before the 4.0 release.